packages/core/src/App/Containers/AccountSwitcherWallet/account-switcher-wallet.tsx

```diff
@@ -39,6 +46,12 @@ export const AccountSwitcherWallet = observer(({ is_visible, toggle }: TAccountS
     useOnClickOutside(wrapper_ref, closeAccountsDialog, validateClickOutside);
 
     const handleTradersHubRedirect = async () => {
+        if (isHubRedirectionEnabled && !!trading_hub) {
+            const is_production = window.location.hostname === PRODUCTION_URL;
+            const redirect_url = is_production ? PRODUCTION_REDIRECT_URL : STAGING_REDIRECT_URL;
+            window.location.assign(redirect_url);
```

Comment:
Instead of redefining the urls here could the urls from platform-config.ts which already has the check for production be used instead 🤔 
```suggestion
            window.location.assign(platforms.tradershub_os.url);
```